langs/c/compile.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -ex
+
+docker build -t llvm - <<EOF
+  FROM ubuntu
```

Comment:
I apparently had an "ubuntu" image laying around, because I ended up with an image with llvm-6 on it, so better be explicit, this worked for me:
```suggestion
  FROM ubuntu:20.04
```